tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/Process/Options/PythonProcessOptions.cs

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.Runtime.InteropServices;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.Sdk.Tools.Cli.Helpers;
+
+/// <summary>
+/// Process options for running Python executables with automatic virtual environment resolution.
+/// Resolves Python executables from AZSDKTOOLS_PYTHON_VENV_PATH environment variable.
+/// </summary>
+public class PythonProcessOptions : ProcessOptions
+{
+    private const string VenvEnvironmentVariable = "AZSDKTOOLS_PYTHON_VENV_PATH";
+
+    /// <summary>
+    /// Creates process options for a Python executable with automatic venv resolution.
+    /// </summary>
+    /// <param name="executableName">Name of the Python executable (e.g., "python", "pytest", "azpysdk")</param>
+    /// <param name="args">Command line arguments</param>
+    /// <param name="logger">Optional logger for resolution diagnostics</param>
+    /// <param name="workingDirectory">Working directory for the process</param>
+    /// <param name="timeout">Execution timeout</param>
+    /// <param name="logOutputStream">Whether to log stdout/stderr</param>
+    public PythonProcessOptions(
+        string executableName,
+        string[] args,
+        ILogger? logger = null,
```

Comment:
Mind swapping `logger` and `workingDirectory` here? They just seem to group together, more naturally.